docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.3'
+services:
+  banaan:
```

Comment:
Gebruik aub descriptive names, niet banaan....

Reply:
Resolved: file is deleted